<?xml version="1.0"?>
<launch>
  <arg name="config" default="$(find ca_driver)/config/default.yaml" />
  <arg name="desc" default="true" />

  <node name="ca_driver" pkg="ca_driver" type="ca_driver" output="screen">
    <rosparam command="load" file="$(arg config)" />
    <param name="robot_model" value="CREATE_1" />
    <param name="estimate_odom_with_cmd_vel" value="true" />
  </node>

  <!-- Robot description -->
  <include if="$(arg desc)" file="$(find ca_description)/launch/create_1.launch" />
</launch>
